File: src/main/java/love/simbot/example/tools/ImageUtil.java
package love.simbot.example.tools;

import cn.hutool.core.codec.Base64Decoder;

public class ImageUtil {
    public static byte[] toImage(String base64) {
        try {
            byte[] data = Base64Decoder.decode(base64);
            for (int i = 0; i < data.length; i++) {
                if (data[i] < 0) {
                    data[i] += 256;
                }
            }
            return data;
        }
        catch (Exception e) {
            return null;
        }
    }
}

File: src/main/kotlin/tools/ImageUtil.kt
package tools

import cn.hutool.core.codec.Base64Decoder

object ImageUtil {
    fun toImage(base64: String?): ByteArray? {
        return try {
            val data = Base64Decoder.decode(base64)
            for (i in data.indices) {
                if (data[i] < 0) {
                    data[i]=(data[i].toInt() + 256).toByte()
                }
            }
            data
        } catch (e: Exception) {
            null
        }
    }
}